canonical-web-and-design/vanilla-framework
# Side navigation variant for Discourse markup

## Done

Adds a new variant to side navigation pattern that supports HTML markup without class names (for use with Discourse based docs).

Fixes #3021 

## QA

- ./run or [demo](https://vanilla-framework-canonical-web-and-design-pr-3052.run.demo.haus/)
- Check new [discourse variant of side-navigation](https://vanilla-framework-canonical-web-and-design-pr-3052.run.demo.haus/docs/examples/patterns/side-navigation/discourse)
- make sure it renders as expected, and works with mobile toggle etc just like standard side navigation
- make sure other existing variants of side navigation still work as before
- Check the updated [side navigation documentation](https://vanilla-framework-canonical-web-and-design-pr-3052.run.demo.haus/docs/patterns/navigation#raw-html)
  - NOTE: CodePen contents are served with published version of Vanilla, so don't render latest styles
- Check the updated [documentation layout docs](https://vanilla-framework-canonical-web-and-design-pr-3052.run.demo.haus/docs/layouts/documentation#content-area)

Demo of this side navigation on discourse docs site: [Snapcraft POC demo](https://snapcraft-io-canonical-web-and-design-pr-2740.run.demo.haus/docs)
